content/en/blog/2024/kubecon-eu.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Join us for OpenTelemetry Talks and Activities at KubeCon EU 2024
+linkTitle: KubeCon EU '24
+date: 2024-02-28
+# prettier-ignore
+cSpell:ignore: Aiven Alexandre Anusha Arbiv Beemer Benedikt Blanco Bongartz Chekuri Coralogix Cosmonic Dyrmishi Jiekun Joonas Kanal Kolachala Kowall Machado Magno Marcin Matej Mirabella Narapureddy Nenashev Oleg Oluwalolope Outshift Pismo Purvi Quwan Reddy Ridwan Rollouts Ryanair Skyscanner Sodkiewicz Soluções Srikanth Tecnológicas Yosef
+author: '[Severin Neumann](https://github.com/svrnm) (Cisco)'
+---
+
+The OpenTelemetry project maintainers, members of the governance committee, and
+technical committee are thrilled to be at [KubeCon EU][] in Paris from March
```

Comment:
```suggestion
technical committee are thrilled to be at [KubeCon + CloudNativeCon Europe][] in Paris from March
```

Reply:
"KubeCon + CloudNativeCon Europe" is the actual name of the conference :-)

Reply:
Should we also include Observability Day, co-located with KubeCon?

Reply:
Good call out! Yes, we can have Observability Day as well

Reply:
```suggestion
technical committee are thrilled to be at [KubeCon + CloudNativeCon Europe][] and at the co-located [Observability Day](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/observability-day/) in Paris from March
```